test/system/030-run.bats

```diff
@@ -1067,16 +1133,18 @@ EOF
     # Run two containers with the same port bindings. The second must fail
     run_podman     run -p $port:80 --rm -d --name $c_ok           $IMAGE top
     run_podman 126 run -p $port:80      -d --name $c_fail_no_rm   $IMAGE top
+    assert "$output" =~ "ddress already in use"
     run_podman 126 run -p $port:80 --rm -d --name $c_fail_with_rm $IMAGE top
+    assert "$output" =~ "ddress already in use"
     # Prior to #15060, the third container would still show up in ps -a
-    run_podman ps -a --sort names --format '{{.Image}}--{{.Names}}'
-    is "$output" "$IMAGE--${c_ok}
-$IMAGE--${c_fail_no_rm}" \
+    run_podman ps -a --sort names --format '--{{.Image}}-{{.Names}}--'
+    assert "$output" =~ "--$IMAGE-${c_ok}--.*--$IMAGE-${c_fail_no_rm}--" \
```

Comment:
This looks fragile, the check is supposed to ensure ctr3 is gone but now we just match part of it.

I guess it would still fail due the leak check at the end but this would be a confusing error for anyone looking at the logs as it is not obvious there what the issue was. I think this could need a `$output !~ $c_fail_with_rm` check

Reply:
Oof. Nice catch, thank you.